### ISSUE TITLE
vue cli can't work

### ISSUE THREAD

**Issue by hiro05097952** [aa910113/ImreThematic#2]
## What problem does this feature solve?
I really like vue-cli for its ease of use while still being highly customizable. My team has integrated vue-cli smoothly into our workflow using a custom plugin. This process is working so great that we have started experimenting with using it for non-Vue projects as well. However, since Vue is always automatically installed as a dependency, we have to remove it manually after install, which is time-consuming.

In short, we would like a command-line option to not automatically install vue, to be able to use vue-cli for non-vue projects.

## What does the proposed API look like?
vue create would get a new option that could be called --no-vue. If present, cli-service generator would skip adding vue and vue-template-compiler as dependencies.

I am willing to help implement this feature if it is found to be within the scope of this project.
